types/alexa-sdk/index.d.ts

```diff
@@ -154,18 +251,682 @@ export interface OutputSpeech {
+     * REPLACE_ALL: Immediately begin playback of the specified stream, and replace current and enqueued streams.
+     * ENQUEUE: Add the specified stream to the end of the current queue. This does not impact the currently playing stream.
+     * REPLACE_ENQUEUED: Replace all streams in the queue. This does not impact the currently playing stream.
+     * @param url Identifies the location of audio content at a remote HTTPS location.
+     * The audio file must be hosted at an Internet-accessible HTTPS endpoint. HTTPS is required, and the domain hosting the
+     * files must present a valid, trusted SSL certificate. Self-signed certificates cannot be used.
+     * The supported formats for the audio file include AAC/MP4, MP3, HLS, PLS and M3U. Bitrates: 16kbps to 384 kbps.
+     * @param token A token that represents the audio stream. This token cannot exceed 1024 characters
+     * @param expectedPreviousToken A token that represents the expected previous stream.
+     * This property is required and allowed only when the playBehavior is ENQUEUE. This is used to prevent potential race conditions
+     * if requests to progress through a playlist and change tracks occur at the same time.
+     * @param offsetInMilliseconds The timestamp in the stream from which Alexa should begin playback.
+     * Set to 0 to start playing the stream from the beginning. Set to any other value to start playback from that associated point in the stream
+     * @returns ResponseBuilder
+     */
+    audioPlayerPlay(behavior: string, url: string, token: string, expectedPreviousToken: string, offsetInMilliseconds: number): ResponseBuilder;
+
+    /**
+     * Creates an AudioPlayer Stop directive - Stops the current audio Playback
+     * @returns ResponseBuilder
+     */
+    audioPlayerStop(): ResponseBuilder;
+
+    /**
+     * Creates an AudioPlayer ClearQueue directive - clear the queue without stopping the currently playing stream,
+     * or clear the queue and stop any currently playing stream.
+     * @param clearBehavior Describes the clear queue behavior. Accepted values:
+     * CLEAR_ENQUEUED: clears the queue and continues to play the currently playing stream
+     * CLEAR_ALL: clears the entire playback queue and stops the currently playing stream (if applicable).
+     * @returns ResponseBuilder
+     */
+    audioPlayerClearQueue(clearBehavior: string): ResponseBuilder;
+
+    /**
+     * Creates a Display RenderTemplate Directive
+     * Use a template builder to generate a template object
+     * @param template template
+     * @returns ResponseBuilder
+     */
+    renderTemplate(template: Template): ResponseBuilder;
+
+    /**
+     * Creates a hint directive - show a hint on the screen of the echo show
+     * @param hintText text to show on the hint
+     * @param hintType (optional) Default value : PlainText
+     * @returns ResponseBuilder
+     */
+    hint(hintText: string, hintType: HintType): ResponseBuilder;
```

Comment:
The JSDoc say hintType is optional but the parameter does not have an optional qualifier.

Reply:
Fixed.